elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/dialects/impl/PrestoDialect.java

```diff
@@ -20,4 +20,10 @@ public String getDialectType() {
     public boolean useAliasForOrderByClause() {
         return true;
     }
+
+    @Override
+    public String appendOffsetLimit(String sql, int offset, int limit) {
+        // offset is supported in prestosql but not in prestodb
```

Comment:
May be the dialect should be renamed to PrestoDB dialect to highlight the fact that we have 2 flavors of Presto?

Reply:
renamed to PrestoDB